javascript/javascript1/week4/character_frequency/character.js

```diff
@@ -0,0 +1,20 @@
+const characters = [];
+function getCharacterFrequencies(inputString) {
+    for (var i = 0; i < inputString.length; i++) {
+        let freq = new Object();
+        let char = inputString.charAt(i);
+        let objectInArray = characters.find(item => item.character === char);
+        if (objectInArray) {
```

Comment:
OK `objectInArray` is assigned but not used anywhere. So what is it actually doing? Something is wrong here and i doubt this code works.